Sources/Networking/HTTPClient/HTTPClient.swift

```diff
@@ -207,7 +214,7 @@ extension HTTPClient: @unchecked Sendable {}
 
 // MARK: - Private
 
-private extension HTTPClient {
+internal extension HTTPClient {
```

Comment:
Modified the access modified here so I could do some testing

Reply:
Would using the `@Testable import RevenueCat` thing have made this visible for testing? I can't _exactly_ remember the correc usecase for it but figured I'd ask 😇

Reply:
I wish! Unfortunately `@testable` only grants you access to internal stuff, not private

Reply:
honestly any time we're making private stuff internal for testing, it's a sign that it should likely be extracted into a separate internal entity instead. Not only for easier testing, but also to reduce the amount of stuff a single object (in this case, HTTPClient) is doing